pkg/client/src/app/pages/repositories/Mvn.tsx

```diff
@@ -36,7 +104,7 @@ export const RepositoriesMvn: React.FunctionComponent = () => {
       </PageSection>
       <PageSection>
         <Card>
-          <CardBody>
+          {/* <CardBody>
```

Comment:
Is this needed ?

Reply:
I will add a TODO for a stretch goal for this. As jeff mentioned.